screepsapi/screepsapi.py

```diff
@@ -33,9 +33,15 @@ def req(self, func, path, **args):
     def get(self, _path, **args): return self.req(requests.get, _path, params=args)
     def post(self, _path, **args): return self.req(requests.post, _path, json=args)
 
-    def __init__(self, u=None, p=None, ptr=False):
+    def __init__(self, u=None, p=None, ptr=False, host=None, secure=False):
         self.ptr = ptr
+        self.host = host
+        self.secure = secure
         self.prefix = 'https://screeps.com/ptr/api/' if ptr else 'https://screeps.com/api/'
+        if host is not None:
+            self.prefix = 'https://' if secure else 'http://'
+            self.prefix += host + '/api/'
```

Comment:
Could you make this an if/else statement and move the `self.prefix` into the else statement?